cmd/sovereignnode/chainSimulator/tests/bridge/incoming_test.go

```diff
@@ -126,10 +120,9 @@ func generateRandomHash() []byte {
 	return randomBytes
 }
 
-func createTransactionsEvent(receiver []byte) []*transaction.Event {
-	codec := createDataCodec()
-	tokenData, _ := codec.SerializeTokenData(createTokenData())
-	eventData, _ := codec.SerializeEventData(createEventData())
+func createTransactionsEvent(nodeHandler process.NodeHandler, receiver []byte, token string, amountToTransfer string) []*transaction.Event {
```

Comment:
I think here we can pass `DataCodecHandler`, since that is the only thing we need from `nodeHandler`